### DISCUSSION TITLE
Empty configuration for Fibaro motion sensor FGMS001

### DISCUSSION THREAD

**Discussion by bostjane** [zwave-js/zwave-js#6505] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Configuration panel for Fibaro motion sensor FGMS001 is empty. Also assigning groups is not possible as it is empty. I tried to manually put device's json file to config folder and made no difference. Re-interview of woken device doesn't help.

What did you expect to happen instead?
I would expect populated configuration panel as in device database (https://devices.zwave-js.io/?jumpTo=0x010f:0x0800:0x1001:0.0#)

### Software versions

Z-Wave JS UI: 9.4.0

### Device information

Manufacturer: Fibaro
Model name: Motion sensor FGMS001
Node ID: 7

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2023-11-20.log](https://github.com/zwave-js/node-zwave-js/files/13406994/zwavejs_2023-11-20.log)

**Comment by AlCalzone** [comment]
Do you have a custom config file for that one?
```
Error parsing config file fgms001.json: Could not parse config file D:\zwave-j
                                  s-ui\store\config\templates\fibaro_template.json: ENOENT: no such file or dire
                                  ctory, open 'D:\zwave-js-ui\store\config\templates\fibaro_template.json'
```

**Comment by bostjane** [comment]
Oh, I haven't realized before that I need that file. Anyway that was my last resort in the first place.
I checked device and am I wrong if device is in the https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x010f/fgms001.json doesn't that mean it's supported out of the box?
If that is true, shouldn't I see configuration panel when I removed /config/fgms001.json?

Here is new log without custom device file: 
[zwavejs_2023-11-20.log](https://github.com/zwave-js/node-zwave-js/files/13418468/zwavejs_2023-11-20.log)

**Reply by AlCalzone** [reply]
Ok I think I wasn't clear. It looks like you have a custom config file in place that isn't valid, because it references a non-existant template. You shouldn't need that at all, just remove the custom file:
D:\zwave-js-ui\store\config\fgms001.json

**Comment by bostjane** [comment]
I understood you well :) Therefore I deleted the file and test it again, no difference. The log is in my previous post.

**Comment by AlCalzone** [comment, marked as answer]
According to the device, it does NOT support Configuration CC nor any of the CCs to manage associations:
```
supported CCs:
· Z-Wave Plus Info
· Version
· Manufacturer Specific
· Association Group Information
· Battery
· Powerlevel
· CRC-16 Encapsulation
· Application Status
· Multilevel Sensor
· Security
· Firmware Update Meta Data
```
I wonder why the Association Group Information CC is in that list though, maybe something in that device is messed up.

Try excluding it, factory resetting and then including again.

**Comment by bostjane** [comment]
Factory reset helped. This is not first Fibaro device that introduced abnormal behavior and needed factory reset.
